src/js/components/modals/AppModalComponent.jsx

```diff
@@ -119,13 +119,33 @@ var AppModalComponent = React.createClass({
   getGeneralErrorBlock: function () {
     var error = this.state.error;
 
+    if (this.state.jsonMode && error != null) {
+      error = Object.keys(error).map(key => {
```

Comment:
This unfortunately produces https://github.com/mesosphere/marathon/issues/3537#issuecomment-199795480